lib/gui/app/modules/analytics.js

```diff
@@ -70,10 +73,7 @@ function getConfig(url) {
  * @param {Object} - config
  */
 // eslint-disable-next-line
-function getMixpanelConfig(config) {
-  const analytics = config.analytics || {}
-  const mixpanel = analytics.mixpanel || {}
-  const probability = mixpanel.probability || DEFAULT_PROBABILITY
+function isClientEligible(mixpanel, probability) {
```

Comment:
The function name seems wrong, also it should only take a mixpanel parameter as probability is included in it.